ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html

```diff
@@ -4,6 +4,10 @@
 #}
 {% extends "package/resource_read.html" %}
 
+{% block wrapper_class %}
+  resource-main
```

Comment:
what is `resource-main`?

Reply:
This is a class that I added to the div containing all of the content on the resource page. The div otherwise uses a set of classes that every other page uses together. `resource-main` is a class so I can target this specific div, to change the display of the page, without messing with the other pages. CKAN helpfully has the `wrapper_class` block to do so.